src/harness/core/functions.ts

```diff
@@ -0,0 +1,3 @@
+namespace core {
+    export function identity<T>(v: T): T { return v; }

```

Comment:
i think we have this in core already.

Reply:
Planning to remove. See https://github.com/Microsoft/TypeScript/pull/20763#discussion_r183806003 for more information.